# gnss_converter
origin: [37.450631,126.6529135,49.451]
#토너먼트
autonomous_goal: [35.069481592976622, 128.57889197025276, 49.0]

# Totla  Use it together
waypoints: [ 
  [37.4483338, 126.6537511, 49.0], #left bottm
  [35.069618508613225, 128.57887159855932, 49.0], #mid up
  [35.069481592976622, 128.57889197025276, 49.0],
]

goal_range: 1.5

servo_range: [78, 113]

thruster: 1550

kp_servo : 0.4
kd_servo : 0.1

# static
angle_number: 10

detecting_angle: 30

margin: 5

filter_queue_size: 1000000

# fuzzy
